tools/istio-iptables/pkg/builder/iptables_builder_impl.go

```diff
@@ -87,7 +87,16 @@ func (rb *IptablesBuilderImpl) AppendRuleV6(chain string, table string, params .
 
 func (rb *IptablesBuilderImpl) BuildV4() []string {
 	rules := []string{}
+	chainTableLookupMap := make(map[string]bool)
```

Comment:
Since you always ignore the boolean value, you can just use `struct{}` to clarify that no value is needed:
```suggestion
	chainTableLookupMap := make(map[string]struct{})
```